lib/src/Base/Stat/SampleImplementation.cxx

```diff
@@ -445,6 +445,18 @@ Bool SampleImplementation::ParseComment(const String & line, const String & mark
 }
 
 
+Bool SampleImplementation::GetLine(std::ifstream & file, String & line)
+{
+  const Bool ok = std::getline(file, line);
+  // deal with DOS EOLs from UNIX
+  if (line.size() && (line[line.size() - 1] == '\r'))
+  {
+    line = line.substr(0, line.size() - 1);
```

Comment:
No need to copy, you can write for instance `line.erase(line.size() - 1, 1)`.

Reply:
Define this function in an anonymous namespace instead of a static method.

Reply:
ok